### ISSUE TITLE
TypeError: Object of type 'Task' is not JSON serializable

### ISSUE THREAD

**Issue by zhangliganggm** [mher/flower#790]
http://localhost:5555/task/
![tim 20180322222855](https://user-images.githubusercontent.com/10146867/37776746-f2577576-2e20-11e8-93b7-8f76d353b30a.png)

http://localhost:5555/api/task/info/uuid
```

Traceback` (most recent call last):
      File "C:\Python364\lib\site-packages\tornado\web.py", line 1541, in _execute
        result = method(*self.path_args, **self.path_kwargs)
      File "C:\Python364\lib\site-packages\tornado\web.py", line 2949, in wrapper
        return method(self, *args, **kwargs)
      File "C:\Python364\lib\site-packages\flower\api\tasks.py", line 605, in get
        self.write(response)
      File "C:\Python364\lib\site-packages\tornado\web.py", line 741, in write
        chunk = escape.json_encode(chunk)
      File "C:\Python364\lib\site-packages\tornado\escape.py", line 81, in json_encode
        return json.dumps(value).replace("</", "<\\/")
      File "C:\Python364\lib\json\__init__.py", line 231, in dumps
        return _default_encoder.encode(obj)
      File "C:\Python364\lib\json\encoder.py", line 199, in encode
        chunks = self.iterencode(o, _one_shot=True)
      File "C:\Python364\lib\json\encoder.py", line 257, in iterencode
        return _iterencode(o, 0)
      File "C:\Python364\lib\json\encoder.py", line 180, in default
        o.__class__.__name__)
    TypeError: Object of type 'Task' is not JSON serializable
```

**Comment by kingofsystem** [comment]
I have the same issue. It appears when i trying to invoke `/api/task/info/`. When i using  `/api/tasks/` all works great.

**Comment by pbsphp** [comment]
Looks like it has been fixed in #719.

**Comment by rafaelreuber** [comment]
Have you installed flower from pip did you get it from master branch?
